src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -271,3 +253,186 @@
         }
     }
 }
+
+.ibexa-table {
+    font-family: $font-family-sans-serif;
+    font-size: $ibexa-text-font-size;
+    line-height: calculateRem(24px);
+    border-spacing: 0;
+    background-color: $ibexa-color-white;
+
+    &__row:nth-child(odd) {
+        background-color: $ibexa-color-light-300;
+    }
+
+    &__cell:first-child {
+        border-top-left-radius: $ibexa-border-radius;
+        border-bottom-left-radius: $ibexa-border-radius;
+    }
+
+    &__cell:last-child {
+        border-top-right-radius: $ibexa-border-radius;
+        border-bottom-right-radius: $ibexa-border-radius;
+    }
+
+    &__cell.ibexa-table__cell {
+        vertical-align: middle;
+
+        .form-check {
+            display: flex;
+            height: calculateRem(16px);
+            width: calculateRem(16px);
+        }
+    }
+
+    &__empty-table-cell {
+        background: $ibexa-color-white;
+        text-align: center;
+    }
+
+    &__empty-table-image {
+        margin-bottom: calculateRem(28px);
+    }
+
+    &__empty-table-text {
+        @include ibexa-font-sans-serif();
+
+        font-size: $ibexa-text-font-size;
+        line-height: calculateRem(24px);
+
+        strong {
+            font-size: $ibexa-text-font-size-large;
+            line-height: calculateRem(33px);
+            font-weight: 600;
+        }
+    }
+
+    &__header-cell {
+        .table & {
+            @include ibexa-font-sans-serif();
+
+            font-size: $ibexa-text-font-size-small;
+            color: $ibexa-color-dark-400;
+            line-height: calculateRem(18px);
+            vertical-align: middle;
+            border-top: calculateRem(1px) solid $ibexa-color-dark-200;
+            border-bottom: 0;
+            padding-top: calculateRem(18px);
+            padding-bottom: calculateRem(18px);
+        }
+
+        .table &--has-icon {
+            max-width: calculateRem(52px);
+        }
+
+        &--field-definitions-head {
+            width: calc(100% / 3);
+        }
+    }
+
+    &__header-cell-checkbox {
+        &.ibexa-input--checkbox {
+            margin-bottom: 0;
+        }
+    }
+
+    &__sort-column {
+        position: relative;
+        cursor: pointer;
+        display: inline-block;
```

Comment:
Note: this one is important, it fixes arrow position for multiline header cell:

| before | after |
|-------|-----|
| ![Screenshot 2021-08-05 at 16 59 27](https://user-images.githubusercontent.com/10233057/128373382-e3520336-625d-4f54-b4a7-7be19b9d977e.png) | ![Screenshot 2021-08-05 at 16 59 19](https://user-images.githubusercontent.com/10233057/128373415-4915b5fe-4c8c-42af-bc0e-2fc45f5fede3.png) |